compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -10272,8 +10477,11 @@ TR::Node *iushrSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s
       }
 
    normalizeConstantShiftAmount(node, INT_SHIFT_MASK, secondChild, s);
-   BINARY_IDENTITY_OP(Int, 0)
 
+   auto binOpSimplifier = getShortBinaryOpSimplifier(s);
```

Comment:
The original code calls `BINARY_IDENTITY_OP(Int, 0)` which calls `secondChild->getIntType()`. Should the corresponding new API be `getIntBinaryOpSimplifier`?